### PR TITLE
BFS Traversal in Binary Tree

### PR DESCRIPTION
**Pull request by pallavidhere23**
Issue #243 bfs traversal is solved here.

### CODE REVIEW

**Comment by Rohan27s** [comment]
@pallavidhere23 Firstly remove the author name it is not required, then decrease the double line spaces to single line spaces and at last mention INPUT and OUTPUT above their respective values

File: Code/Java/Bfs_Traversal.java [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+2) temp_node = root //start from root
+3) Loop while temp_node is not NULL
+    a) print temp_node->data.
+    b) Enqueue temp_node�s children 
+      (first left then right children) to q
+    c) Dequeue a node from q.*/
+
+ 
+/**
+ * @author Pallavi Dhere
```
Comment (Rohan27s) [inline Code/Java/Bfs_Traversal.java:18]: Remove this

File: Code/Java/Bfs_Traversal.java [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        arr=new int[t];
+        for(int i=0;i<t;i++) {
+            arr[i]=Integer.parseInt(br.readLine());
+          }
+        
+        t2.root = t2.insertLevelOrder(arr, t2.root, 0); 
+        t2.bfs(t2.root); 
+    } 
+}
+/*INPUT AND OUTPUT
```
Comment (Rohan27s) [inline Code/Java/Bfs_Traversal.java:116]: The heading Must be Test Case 1

File: Code/Java/Bfs_Traversal.java [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+ *Enter the Number of notes in a tree
+5
```
Comment (Rohan27s) [inline Code/Java/Bfs_Traversal.java:118]: Write INPUT before mentioning input values and same for the output

File: Code/Java/Bfs_Traversal.java [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+printLevelorder(tree)
+1) Create an empty queue q
+2) temp_node = root //start from root
+3) Loop while temp_node is not NULL
+    a) print temp_node->data.
+    b) Enqueue temp_node�s children 
+      (first left then right children) to q
+    c) Dequeue a node from q.*/
+
```
Comment (Rohan27s) [inline Code/Java/Bfs_Traversal.java:15]: Remove unnecessary space

File: Code/Java/Bfs_Traversal.java (hunk 1 above)
Comment (Rohan27s) [inline Code/Java/Bfs_Traversal.java:18]: Remove this

**Comment by Rohan27s** [comment]
There is one compilation error in the code 
-class Bfs_Traversal is public, should be declared in a file named Bfs_Traversal.java
public class Bfs_Traversal 

Kindly resolve this issue

**Review by Rohan27s — COMMENTED** [review]
Resolve all the conflicts

**Comment by pallavidhere23** [comment]
@Rohan27s , I have made all the requested , kindly review it.

**Comment by Rohan27s** [comment]
@pallavidhere23 Resolve the compilation error, the error is still occurring

**Comment by pallavidhere23** [comment]
@Rohan27s ,when I am running in my desktop its working fine. Can u please share snippets of errors.

**Comment by Rohan27s** [comment]
image

**Comment by Rohan27s** [comment]
If it is working on your pc then send me the output's snippet

**Comment by pallavidhere23** [comment]
Screenshot (31)

**Comment by pallavidhere23** [comment]
@Rohan27s , I have uploaded the snippet. Kindly check.

**Comment by Rohan27s** [comment]
image

**Comment by Rohan27s** [comment]
Try to run it on an online compiler and then send me the snippet

**Comment by pallavidhere23** [comment]
> Try to run it on an online compiler and then send me the snippet
In online compiler its giving error .

**Comment by Rohan27s** [comment]
> > Try to run it on an online compiler and then send me the snippet
> > In online compiler its giving error .

Resolve it

**Comment by pallavidhere23** [comment]
@yasharth291 , Sorry for the delay , My Semester End Exams was going on. Please if possible activate this pull request. I'll Finished it asap.

**Pull request closed without merging**